translation/v2_to_v3.xsl

```diff
@@ -31,9 +31,9 @@ Version: 0.1.0
     </xsl:copy>
   </xsl:template>
 
-  <xsl:template match="@xsi:schemaLocation">
+  <!--xsl:template match="@xsi:schemaLocation">
     <xsl:attribute name="xsi:schemaLocation">http://buildingsync.net/schemas/bedes-auc/2019 https://raw.githubusercontent.com/BuildingSync/schema/v3.0.0/BuildingSync.xsd</xsl:attribute>
-  </xsl:template>
+  </xsl:template-->
```

Comment:
I'm ok with this for now, but maybe we should tweak our examples so that we can allow this to be included. e,g, if a user has a schemalocation that points to an older version then it'll be incorrect